components/about/MissionDetail.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+interface PropsInterface {
+    className?: string
+    title: string
+    info: string
+}
+
+export default function MissionDetail(props: PropsInterface) {
+    const { className, title, info } = props;
+
+    return (
+        <div className={`flex flex-col md:flex-row justify-center items-center md:mr-[5vw] ${className}`}>
+            <h3 className='flex justify-center md:justify-end md:mr-10 font-bold w-[70vw] md:w-[35vw] text-4xl mb-3'>{title}</h3>
+            <p className='text-center md:text-left w-[70vw] md:w-[30vw] text-lg'>{info}</p>
+        </div>
```

Comment:
mao ni ang nice abstraction very nice

Reply:
ig implement nato sa cms kay ato ra i map with an array of data very very nice i love u max